client/package.json

```diff
@@ -151,6 +151,7 @@
 		"express-async-handler": "^1.1.4",
 		"express-useragent": "^1.0.15",
 		"fake-indexeddb": "^3.1.8",
+		"fast-deep-equal": "^3.1.3",
```

Comment:
👍 Nice job re-using the same package and version we're already transitively dependent on.